source/index.html.md

```diff
@@ -1360,7 +1361,8 @@ const secret = "";
 
 
 const body = {
-	"id": "qwd19992-43fd-14e8-b027-bb815bnb14ed", //Replace it with your Order ID.
+	// "id": "qwd19992-43fd-14e8-b027-bb815bnb14ed", //Replace it with your Order ID.
+	"client_order_id": "qwd19992-43fd-14e8-b027-bb815bnb14ed", //Replace it with your Client Order ID.
```

Comment:
Let's use `2022.02.14-btcinr_1` only as the example value for `client_order_id` throughout the page - uniformity helps to follow better.

Reply:
[Optional] Also, let's prefer to use `axxxxxxa-axxa-axxa-axxa-axxxxxxxxxxa` as the example order.id (at least for the new additions; while existing ones can be looked at later).